addons/crm/static/src/js/crm_kanban.js

```diff
@@ -4,12 +4,156 @@
+            // Check whether the given MRR sum field is valid or not
+            this.isRecurringRevenueSumFieldValid = Object.keys(columnState.fields).includes(columnState.progressBarValues.recurring_revenue_sum_field);
+            this.recurringRevenueSumFieldLabel = this.isRecurringRevenueSumFieldValid ? columnState.fields[this.recurringRevenueSumField].string : false;
+            // Previous progressBar state
+            const state = options.progressBarStates[this.columnID];
+            if (state) {
+                this.totalRecurringRevenue = state.totalRecurringRevenue;
+            }
+        },
+
+        /**
+         * @override
+         */
+        willStart() {
+            const userHasGroup = this.getSession().user_has_group('crm.group_use_recurring_revenues')
+                .then((useRecurringRevenues) => {
+                    // only show MRR related info if 'Recurring Revenues' is enabled, and provided MRR sum field is valid
+                    this.showRecurringRevenue = useRecurringRevenues && this.isRecurringRevenueSumFieldValid;
+                });
+            return Promise.all([this._super(...arguments), userHasGroup]);
+        },
+
+        /**
+         * @override
+         */
+        start() {
+            this._super.apply(this, arguments);
+            if (this.showRecurringRevenue) {
+                this.$mrrCounter = this.$counter.filter('.o_crm_kanban_mrr_counter_side');
+                this.$recurringRevenueNumber = this.$mrrCounter.find('.o_crm_kanban_mrr_sum');
+                // the MRR counter has it's own width, and so if we put the '+' symbol outside it,
+                // '+' will be more close to the actual sum field and far from MRR, especially if
+                //  MRR has single digit value and text is aligned right, so we put it within MRR.
+                let $plus = $('<strong/>', {
+                    text: '+',
+                });
+                this.$('.o_kanban_counter_progress').addClass('o_crm_kanban_counter_progress_mrr');
+                this.$mrrCounter.prepend($plus);
+            }
+        },
+
+        /**
+         * @override
+         */
+        computeCounters() {
+            this._super.apply(this, arguments);
+            if (this.showRecurringRevenue) {
+                this.prevTotalRecurringRevenue = this.totalRecurringRevenue;
+                this.totalRecurringRevenue = this.columnState.aggregateValues[this.recurringRevenueSumField] || 0;
+            }
+        },
+
+        //--------------------------------------------------------------------------
+        // Private
+        //--------------------------------------------------------------------------
+
+        /**
+         * @private
+         * @override
+         */
+        _render() {
+            const self = this;
+            this._super.apply(this, arguments);
+
+            if (this.showRecurringRevenue) {
```

Comment:
Would be easier with an inverse if (aka: if not using, early return). That way we gain an indent level.